WebDriverAgentLib/Commands/FBOrientationCommands.m

```diff
@@ -50,7 +50,8 @@ + (NSArray *)routes
 + (id<FBResponsePayload>)handleGetOrientation:(FBRouteRequest *)request
 {
   FBSession *session = request.session;
-  return FBResponseWithStatus(FBCommandStatusNoError, [self.class interfaceOrientationForApplication:session.activeApplication]);
+  NSString *orientation = [self.class interfaceOrientationForApplication:session.activeApplication];
+  return FBResponseWithStatus(FBCommandStatusNoError, [[self _wdOrientationsMapping] valueForKey:orientation]);
```

Comment:
note that this is a breaking change, but a breaking change to bring into line with the JSONWP. java client used to blanket return "unknown" for anything that wasn't PORTRAIT or LANDSCAPE anyway